backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -250,8 +256,18 @@ func initDBClient(initConnectionTimeout time.Duration) *storage.DB {
 		&model.ResourceReference{},
 	)
 
-	if response.Error != nil {
-		glog.Fatalf("Failed to initialize the databases.")
+	if ignoreAlreadyExistError(response.Error) != nil {
+		glog.Fatalf("Failed to initialize the databases. Error: %s", response.Error)
+	}
+
+	var textFormat string
+	switch driverName {
+	case "mysql":
+		textFormat = "longtext not null"
+	case "pgx":
+		textFormat = "text"
```

Comment:
We might need to define some `const` for value like `text` and `longtext not null`.

Reply:
Could you help me understand why it is better to use `const` values here? Thank you.

Reply:
DB dialects should be stored in a centralized place, so it helps reviewing, modifying and extending. For example: If we integrate with a new DB type in the future, I hope that I can view the existing syntax differences easily in one place, instead of trying to catch those adhoc variables during implementation. 

Therefore, my recommendation is to define a const for `MYSQL_TEXT_FORMAT` and `PGX_TEXT_FORMAT` in a centralized file: https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/storage/db.go

Reply:
Thank you so much for the thourought explanation! I added the constants in `backend/src/apiserver/client/sql.go` instead, because this is a SQL related dependency that client_manager is already using.